### ISSUE TITLE
astroid.modutils.load_module_from_file failing on path when using multiple symlinks

### ISSUE THREAD

**Issue by mariocj89** [PyCQA/astroid#583]
### Steps to reproduce
1. Create a folder and symlinked it to another one. Create a module within it.
```
mkdir /tmp/deployment
ln -s /tmp/deployment /tmp/path_to_include
touch /tmp/secret.py
ln -s /tmp/secret.py /tmp/deployment/secret.py
```
2. Validate ```  PYTHONPATH=/tmp/path_to_include python3.6 -c "import secret" ``` works.
3. Astroid wont be able to load it:
```
PYTHONPATH=/tmp/path_to_include python3.6 
>>> import astroid
>>> astroid.modutils.load_module_from_file('/tmp/path_to_include/secret.py')
```

Note `/tmp/path_to_include/secret.py` exists and we are passing `/tmp/path_to_include` in the path, but as the module resolves it to realpath `load_module_from_file` wont work.

### Current behavior

Astroid is unable to load a module it the python path where it is present is on is a symlink (the one on sys.path, not the file path itself)

### Expected behaviour

To be able to import it the same way you would with python.

### ``python -c "from astroid import __pkginfo__; print(__pkginfo__.version)"`` output

2.0.0.dev3

### version information:

Present in the latest versions of 1.6 and 1.5.
Works in the latest version of 1.4: 1.4.9

Related issue: https://github.com/PyCQA/astroid/issues/538

Related commit: https://github.com/PyCQA/astroid/commit/788bb41c9eb1078237724933e3d33ca98419c41f

This manifests as well for the `extra-path` argument on the same function in modutils.

Proposal: Lookup on both realpath and abspath the same way #538 was resolved.

**Comment by mariocj89** [comment]
I’m trying to put a test case through and will send a PR with the proposal unless you think it is not worth fixing or not a real issue.

**Comment by mariocj89** [comment]
Fixed via #584
